src/components/internals/create-select-styles.js

```diff
@@ -153,6 +153,11 @@ const groupHeadingStyles = () => base => ({
   },
 });
 
+const menuPortalStyles = () => base => ({
+  ...base,
+  zIndex: 101,
```

Comment:
This is how I got it to work locally with the app kit. 

Should we make this configurable? 🤔

Reply:
Should we do this here or do it in the Modal component?

Reply:
I fail to judge pros and cons so can't give a opinions :(

Reply:
[Here](https://github.com/JedWatson/react-select/blob/44e9fb29b230e49a754a2f0d6f30c2250aa45009/src/components/Menu.js#L468) is the default menuPortalStyles

Z-index is set to 1, and this doesn't play nicely with react-modal 😢

Reply:
We should make it configurable yes, because there can be many stacked modals and we don’t know the zindex.

Reply:
let me push another commit

Reply:
Like [this](https://github.com/commercetools/ui-kit/pull/682/commits/72f02ae04d246247c3e137ebbe5530e447b7b9ec)?